plugins/crd/plugin_impl_crd.go

```diff
@@ -77,8 +79,8 @@ type Deps struct {
 	Log  logger.FieldLoggerPlugin
 	// Kubeconfig with k8s cluster address and access credentials to use.
 	KubeConfig  string
-	Handler     handler.PluginAPI
-	ObjectStore objectstore.PluginAPI
+	Handler     handler.API
```

Comment:
Handler plugin has not been used in CRD plugin, if you check the code it is only defined here and used in tests, no other references. I would suggest to remove it from CRD plugin completely.

Reply:
@sbezverk Happy too.   Can we do it in a separate PR?